frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -124,6 +125,7 @@ export interface DockerData {
 }
 
 export interface RouteData {
+  show?: boolean;
```

Comment:
Isn't this always there? Or does `RouteData` get used by an interface that doesn't have this functionality yet?

Reply:
So we need to show the create route option only when the route was not created when the application was created. If a route already exists we need not show the create route option. Now according to new UX we need to disable the option and not hide it. Aligning with new UX in my current story.